packages/TorneloScoresheet/src/pages/ResultDisplay/ResultDisplay.tsx

```diff
@@ -21,8 +22,25 @@ const ResultDisplay: React.FC = () => {
   }`;
 
   const handleEmailGame = (): void => {
-    // TODO: Implement email logic
-    console.log('email game button pressed');
+    const subject =
+      'Tornello Game Results ' +
+      infoString +
+      ':' +
+      resultDisplayMode?.pairing.players[0].firstName +
```

Comment:
The pgn already has thing information so since it's in the subject we might not need to include it here? It doesn't really matter though
<img width="321" alt="image" src="https://user-images.githubusercontent.com/64511606/185394824-1b7600c5-dba4-4938-bbac-69eed052be3f.png">